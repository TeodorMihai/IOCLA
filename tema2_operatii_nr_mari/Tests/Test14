-9999FA221234AFDB * -9999DFFF
