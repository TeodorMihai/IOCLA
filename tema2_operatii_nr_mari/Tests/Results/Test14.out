DBAF341222FA9999
FFDF9999
25F08D04633482FCEA59295C
5C2959EAFC823463048DF025
